sycl/test/basic_tests/query.cpp

```diff
@@ -0,0 +1,43 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
I am not sure this test is expected to work for every target, so should it go to llvm-test-suite instead?